docs/regression-test/test-repos.csv

```diff
@@ -0,0 +1,26 @@
+﻿page view,repos,version,loc,xref,conceptual,SDP conceptual,SDP Ref,"multi
```

Comment:
The first column is not the actual page view number, but rather the page view rank